apps/dialtone-documentation/docs/.vuepress/baseComponents/TokenTable.vue

```diff
@@ -1,3 +1,4 @@
+<!-- eslint-disable vuejs-accessibility/mouse-events-have-key-events -->
```

Comment:
> Enforce onmouseover/onmouseout are accompanied by onfocus/onblur. Coding for the keyboard is important for users with physical disabilities who cannot use a mouse, AT compatibility, and screenreader users.


I don't like to disable eslint rules, I think it's ok to add onfocus/onblur events to make sure keyboard navigation is enabled too.

https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/mouse-events-have-key-events.md

Reply:
In this case, the focus over the row is not enabled, so I didn't think it made much difference. But I will add them in case we add keyboard navigation in the future 👍